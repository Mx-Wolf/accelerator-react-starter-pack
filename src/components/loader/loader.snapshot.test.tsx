import renderer from 'react-test-renderer';
import Loader from './loader';

describe('Component: Loader', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Loader />);
    expect(component).toMatchSnapshot();
  });
});
