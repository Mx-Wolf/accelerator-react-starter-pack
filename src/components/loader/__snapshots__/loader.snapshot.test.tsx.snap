// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loader should render correctly 1`] = `
<div
  className="lds-dual-ring"
>
  Loading...
</div>
`;
